README.rst

```diff
@@ -76,11 +76,10 @@ Long story short, copy-paste this into the console:
 
 .. code-block:: bash
 
-    tarantoolctl rocks install cartridge-cli
-    .rocks/bin/cartridge create --name myapp
+    cartridge create --name myapp
```

Comment:
@lenkis could you help here, please.

It's time to update installation instructions for cli 2.0.

I guess we could move `Prerequisites` section from `Contribution` section above `Create your first application`. WDYT?

Reply:
LGTM